paper.md

```diff
@@ -1,53 +1,90 @@
 ---
-title: 'pyuvdata: an interface for astronomical interferometeric datasets in python'
+title: 'pyuvdata v3: an interface for astronomical interferometeric datasets in python'
 tags:
  - radio astronomy
  - uvfits
  - miriad
 authors:
+ - name: Garrett K. Keating
+   orcid: 0000-0002-3490-146X
+   affiliation: 1
  - name: Bryna J. Hazelton
    orcid: 0000-0001-7532-645X
-   affiliation: 1, 2
+   affiliation: 2, 3
+ - name: Matthew Kolopanis
+   orcid: 0000-0002-2950-2974
+   affiliation: 4
+ - name: Steven Murray
+   orcid: 0000-0003-3059-3823
+   affiliation: 4
+ - name: Adam P. Beardsley
+   orcid: 0000-0001-9428-8233
+   affiliation: 5
  - name: Daniel C. Jacobs
    orcid: 0000-0002-0917-2269
    affiliation: 3
+ - name: Nicholas Kern
+   affiliation: 6
+   orcid: 0000-0002-8211-1892
+ - name: Adam Lanman
+   affiliation: 7
+   orcid: 0000-0003-2116-3573
+ - name: Paul La Plante
+   affiliation: 8
+   orcid: 0000-0002-4693-0102
  - name: Jonathan C. Pober
    orcid: 0000-0002-3492-0433
-   affiliation: 4
- - name: Adam P. Beardsley
-   orcid: 0000-0001-9428-8233
-   affiliation: 3
+   affiliation: 9
+ - name: Pyxie Star
+   orcid: XXXX-XXXX-XXXX-XXXX
+   affiliation: 2
 affiliations:
- - name: University of Washington, eScience Institute
+ - name: Center for Astrophysics | Harvard & Smithsonian
    index: 1
- - name: University of Washington, Physics Department
+ - name: University of Washington, eScience Institute
    index: 2
- - name: Arizona State University, School of Earth and Space Exploration
+ - name: University of Washington, Physics Department
    index: 3
- - name: Brown University, Physics Department
+ - name: Arizona State University, School of Earth and Space Exploration
    index: 4
-date: 22 November 2016
+ - name: Winona State University, Physics Department
+   index: 5
+ - name: Massachusetts Institute of Technology, Physics Department
+   index: 6
+ - name: Kavli Institute of Astrophysics and Space Research
+   index: 7
+ - name: University of Nevada, Las Vegas, Department of Computer Science
+   index: 8
+ - name: Brown University, Physics Department
+   index: 9
+date: 2 July 2024
 bibliography: paper.bib
 ---
 
 # Summary
+pyuvdata is an open-source software package that seeks to do provide a well-documented,
+feature-rich interface for many of the different data formats that exist within radio
+interferometry, including support for reading and writing UVH5 [@uvh5], UVFITS
+[@uvfits], MIRIAD [@miriad], and measurement set [@ms] visibility files; and reading of
+FHD [@fhd] and MIR [@mir] visibility save files. Additionally, pyuvdata supports reading
+and writing measurement set calibration tables, and reading of FHD calibration tables.
```

Comment:
Maybe restructure the calibration sentence to mention the other formats we support, i.e. CalH5 -- no memo yet but I might be able to get one together soon, CalFITS -- there is a memo for this.

Maybe also mention UVBeam and UVFlag? This could be brief but I think it's useful to point out they exist.